CMS-Enterprise/sbom-harbor
# fix(timestamp): changed unix timestamp to rfc3339(iso8601)

## Summary

fix(timestamp): changed unix timestamp to rfc3339(iso8601) 

### Added

Function to generate a iso8601 timestamp in platform that uses the original timestamp() function.

### Changed

Now using the new function to generate a timestamp

## How to test

1. Run Snyk SBOM provider
2. Run Snyk Vulnerability Provider
3. Run SDL Report Analytic
4. Open S3 and verify that iso8601 timestamps exist in the Report
